src/Common/Types.ts

```diff
@@ -935,3 +936,7 @@ export interface EnvironmentHelmResult {
 }
 
 export type EnvironmentListHelmResponse = ResponseType<EnvironmentListHelmResult[]>
+
+export interface TippyProps extends TippyJSProps {
```

Comment:
Please change name

Reply:
Changed name to Tooltip as per @eshankvaish 's suggestion